.pylintrc

```diff
@@ -140,7 +140,8 @@ disable=print-statement,
         exception-escape,
         comprehension-escape,
         no-member,
-        no-name-in-module
+        no-name-in-module,
+        import-error
```

Comment:
Ok, somehow, on mac it works, and on mac CI it works too - and honestly i have no idea, because just the same way I import `from parse ...` and smh this works ... i've changed CI to mac, as i dont even have a way to test it on ubuntu.